.changeset/brave-ghosts-buy.md

```diff
@@ -0,0 +1,9 @@
+---
+"wrangler": patch
+---
+
+fix: parsing of node inspector url
+
+This fixes the parsing of the url returned by Node Inspector via stderr which could be received partially in multiple chunks or in a single chunk.
+
+Closes #1226 
```

Comment:
Hmm, for some reason Prettier doesn't like this...

Reply:
fixed, trailing whitespace